app/controllers/api/databases.php

```diff
@@ -1673,12 +1676,77 @@ function updateAttribute(
             throw new Exception(Exception::COLLECTION_NOT_FOUND);
         }
 
-        $attributes = $collection->getAttribute('attributes');
+        $queriesFromRequest = Query::parseQueries($queries);
 
-        $response->dynamic(new Document([
-            'total' => \count($attributes),
-            'attributes' => $attributes
+        // Add type property in query if select query exists and type property doesn't exist as type is required for response model
+        $hasDetailsinQuery = [
+            'selectQuery' => false,
+            'typeAttributeInSelectQuery' => false,
+        ];
+        foreach ($queriesFromRequest as $query) {
+            if ($query->getMethod() ===  Query::TYPE_SELECT) {
+                $hasDetailsinQuery['selectQuery'] = true;
+            }
+            if (\array_search('type', $query->getValues())) {
+                $hasDetailsinQuery['typeAttributeInSelectQuery'] = true;
+            }
+        }
+        $transformedQueries = $queriesFromRequest;
```

Comment:
We can remove this and push directly onto `$queries`

Reply:
fixed this